src/patternfly/patternfly-charts.scss

```diff
@@ -50,3 +51,172 @@ $pf-chart-color-black-200: $pf-color-black-400;
 $pf-chart-color-black-300: $pf-color-black-500;
 $pf-chart-color-black-400: $pf-color-black-600;
 $pf-chart-color-black-500: $pf-color-black-700;
+
+// typography
+$pf-chart-global--FontSize--xs: 12; // Axis Labels
+$pf-chart-global--FontSize--sm: 14;  // Legend
+$pf-chart-global--FontSize--2xl: 24; // Large Numbers
+
+// label
+$pf-chart-global--label--Padding: 10;
+$pf-chart-global--label--stroke--Width: 0;
+
+// Layout Props
+$pf-chart-global--layout--Padding: 50;
+$pf-chart-global--layout--Height: 300;
+$pf-chart-global--layout--Width: 450;
+
+// Stroke
+$pf-chart-global--strokeLineCap: "round";
+$pf-chart-global--strokeLineJoin: "round";
+$pf-chart-global--strokeDasharray: "10, 5";
+
+// Individual Charts
+
+// Area Chart
+$pf-chart-area--Opacity: .4;
+$pf-chart-area--stroke--Width: 2;
+$pf-chart-area--data--BackgroundColor: "charcoal"; // Grayscale theme
+
+// Axis Chart
+$pf-chart-axis--axis--stroke--Width: 1;
+$pf-chart-axis--axis--stroke--Color: "charcoal"; // Grayscale theme
+$pf-chart-axis--axis--BackgroundColor: transparent; // Grayscale theme
+$pf-chart-axis--axis-label--Padding: 100; // I see 25px padding for this in the grayscale theme
+$pf-chart-axis--axis-label--stroke--Color: "transparent"; // material theme
+$pf-chart-axis--grid--BackgroundColor: transparent; // is transparent ok for none?
+$pf-chart-axis--grid--stroke--Color: "blueGrey50"; // material theme
+$pf-chart-axis--grid--BorderColor: transparent;
+$pf-chart-axis--grid--pointerEvents: "painted";
+$pf-chart-axis--tick--BackgroundColor: transparent;
+$pf-chart-axis--tick--Width: 1; // this is 5 in material
+$pf-chart-axis--tick--BorderColor: transparent;
+$pf-chart-axis--tick--stroke--Color: "blueGrey300";
+$pf-chart-axis--tick-label--BackgroundColor: "blueGrey700";
+
+// Bar Chart
+$pf-chart-bar--Width: 10;
+$pf-chart-bar--data--stroke: none;
+$pf-chart-bar--data--BackgroundColor: "charcoal"; // Grayscale theme
+$pf-chart-bar--data--Padding: 8; // Grayscale theme
+$pf-chart-bar--data-stroke--Width: 0; // Grayscale theme
+
+// Box Plot Chart
+$pf-chart-boxplot--max--Padding: 8;
+$pf-chart-boxplot--max--stroke--Color: "charcoal"; // grayscale theme
+$pf-chart-boxplot--max--stroke--Width: 1;
+$pf-chart-boxplot--median--Padding: 8;
+$pf-chart-boxplot--median--stroke--Color: "charcoal"; // grayscale theme
+$pf-chart-boxplot--median--stroke--Width: 1;
+$pf-chart-boxplot--min--Padding: 8;
+$pf-chart-boxplot--min--stroke--Width: 1;
+$pf-chart-boxplot--min--stroke--Color: "charcoal"; // grayscale theme
+$pf-chart-boxplot--q1--Padding: 8;
```

Comment:
Lost the previous comment. @dlabrecq you pointed me [here](https://formidable.com/open-source/victory/docs/victory-box-plot/#q1) but I'm still not sure what q1 is :) I'm just curious if we can give a more accessible name/description for that var instead of something that's victory specific.

Reply:
q1 means Lower Quartile and q3 means Upper Quartile, basically the beginning and end of the median.